docs/en/sql-reference/20-sql-functions/05-datetime-functions/to-timestamp.md

```diff
@@ -25,14 +14,64 @@ See also: [TO_DATE](to-date)
 ```sql
 -- Convert a string or integer to a timestamp
 TO_TIMESTAMP(<expr>)
+```
 
+If given a string, the function extracts a date from the string; If given is an integer, the function interprets the integer as the number of seconds, milliseconds, or microseconds before (for a negative number) or after (for a positive number) the Unix epoch (midnight on January 1, 1970):
```

Comment:
Only support ISO 8601

Reply:
Updated to "If given an ISO 8601 date format string, the function...."